scene/resources/multimesh.cpp

```diff
@@ -219,6 +219,7 @@ void MultiMesh::set_instance_count(int p_count) {
 	ERR_FAIL_COND(p_count < 0);
 	RenderingServer::get_singleton()->multimesh_allocate_data(multimesh, p_count, RS::MultimeshTransformFormat(transform_format), use_colors, use_custom_data);
 	instance_count = p_count;
+	notify_property_list_changed();
```

Comment:
I would remove the array support for multimesh, given its designed to handle millions of elements and this will likely kill the inspector

Reply:
Ok, removed. :)